library/aesni.c

```diff
@@ -43,6 +43,17 @@
 #include <immintrin.h>
 #endif
 
+#if defined(MBEDTLS_ARCH_IS_X86)
+#   if defined(MBEDTLS_COMPILER_IS_GCC)
+#       pragma GCC push_options
+#       pragma GCC target ("pclmul,sse2,aes")
+#       define MBEDTLS_POP_TARGET_PRAGMA
+#   elif defined(__clang__)
+#       pragma clang attribute push (__attribute__((target("pclmul,sse2,aes"))), apply_to=function)
+#       define MBEDTLS_POP_TARGET_PRAGMA
+#   endif
+#endif /* MBEDTLS_ARCH_IS_X86 */
```

Comment:
While it's nice to have the indentation, the other block added doesn't have it, and nor do (AFAICS) any of the other pre-processor blocks in this file, so best to remove